robot.js

```diff
@@ -96,16 +96,36 @@
 
     switch (robot.dir) {
       case 'up':
-        y = y < board.length - 1 ? y + 1 : y;
+        if (y < board.length - 1) {
+          y = y + 1;
+        } else {
+          y;
```

Comment:
This statement has no effect. You can leave out the whole else block. In the case of the ternary operator it was required because you must supply an expression following the `:` symbol.

This same comments applies to the other ternary operators in the `move` function.